integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParserUtil.scala

```diff
@@ -744,6 +747,47 @@ object CarbonSparkSqlParserUtil {
     CarbonAlterTableAddColumnCommand(alterTableAddColumnsModel)
   }
 
+  def describeColumn(
+      databaseNameOp: Option[String],
+      tableName: String,
+      inputFields: java.util.List[String]
+  ): CarbonDescribeColumnCommand = {
+    val (tableSchema, carbonTable) = getSchemaAndTable(databaseNameOp, tableName)
+    val inputColumn = tableSchema.find(_.name.equalsIgnoreCase(inputFields.get(0)))
+    if (!inputColumn.isDefined) {
+      throw new MalformedCarbonCommandException(
+        s"${inputFields.get(0)} not present in schema of table: $tableName")
+    }
+    CarbonDescribeColumnCommand(
+      carbonTable,
+      inputFields,
+      inputColumn.get
+    )
+  }
+
+  def describeShort(
+      databaseNameOp: Option[String],
+      tableName: String
+  ): CarbonDescribeShortCommand = {
+    val (tableSchema, carbonTable) = getSchemaAndTable(databaseNameOp, tableName)
+    CarbonDescribeShortCommand(
+      carbonTable,
+      tableSchema
+    )
+  }
+
+  def getSchemaAndTable(databaseNameOp: Option[String],
+      tableName: String) : (StructType, CarbonTable) = {
+    val sparkSession = SparkSQLUtil.getSparkSession
+    validateTableExists(databaseNameOp, tableName, sparkSession)
```

Comment:
Add a testcase for Describe column on table not exists. Looks like currently, it is not throwing Table not found, instead throws parsing error.

Reply:
Done